src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/ConfigurationDebuggerView.xaml

```diff
@@ -89,6 +108,9 @@
 			</Setter>
 		</Style>
 
+		<x:String x:Key="UpArrowPathData">M0 0L-5 -5L-10 0H0Z</x:String>
```

Comment:
Should UpArrowPathData and DownArrowPathData be define in a separate file?

Reply:
In my opinion if it's only being used for diagnostics, we should keep it here.

Reply:
yeah we want the diagnostics to be self contained so that we can easily grab them and paste them in existing projects that want to add those new features.